docs/assets/examples/raster/script.js

```diff
@@ -7,14 +7,10 @@ const RASTER_TOKEN = '<API_KEY>';
 const mapProxyBaseUrl = '<MAP_SERVER-RASTER-SERVICE_URL>';
 const INJECTION_TYPE = '<INJECTION_TYPE>';
 
-// according to defined B2B authentication principles HEADER or QUERYPARAM
+// according to defined B2B authentication principles QUERYPARAM
 const getAuthObject = () => {
   const tokenProps = {};
-  if (INJECTION_TYPE.toLowerCase() === 'header') {
-    tokenProps.headers = {
-      'X-API-KEY': RASTER_TOKEN
-    };
-  } else if (INJECTION_TYPE.toLowerCase() === 'queryparam') {
+  if (INJECTION_TYPE.toLowerCase() === 'queryparam') {
```

Comment:
if there is only query why do you need the condition?

Reply:
this should be @alebinson decision